NoSql/Models/ProductItem.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace NoSql.Models
+{
+    public class ProductItem
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string? Image { get; set; }
+        public int Price { get; set; }
```

Comment:
Price could have numbers after the dot (.), please use `decimal` or `float` instead.

```suggestion
        public decimal Price { get; set; }
```